.github/workflows/ci.yml

```diff
@@ -117,11 +117,11 @@ jobs:
         run: yarn workspace main build
         env:
           NODE_ENV: production
-          NEXT_PUBLIC_ORIGIN: https://ci.learn.mit.edu
-          NEXT_PUBLIC_MITOL_API_BASE_URL: https://api.ci.learn.mit.edu
+          NEXT_PUBLIC_ORIGIN: https://cifake.learn.mit.edu
+          NEXT_PUBLIC_MITOL_API_BASE_URL: https://api.cifake.learn.mit.edu
```

Comment:
https://ci.learn.mit.edu is, at time of writing, a  heroku app, though `https://api.ci.learn.mit.edu` doesn't seem to be 

Intent here was not to hit actual servers.

This change makes the CI nextjs build take about 90 seconds, which it had been previously compared to:
- timeouts on `main` during CI https://github.com/mitodl/mit-learn/actions/runs/14804003164/job/41659631667
- 8+ minutes here, where I increased the timeout https://github.com/mitodl/mit-learn/actions/runs/14840635669/job/41663769998